admin/replication/LoadReplicationChanges

```diff
@@ -331,11 +332,17 @@ if (-x "$FindBin::Bin/hooks/post-process") {
 # Also last_replication_date should match TIMESTAMP
 {
     my $row = $sql->select_single_row_hash('SELECT * FROM replication_control');
-    if ($row->{current_replication_sequence} != $REPLICATION_SEQUENCE) {
-        die "Applied changes, but current_replication_sequence is $row->{current_replication_sequence} not $REPLICATION_SEQUENCE\n";
-    }
-    if (defined($TIMESTAMP) && $row->{last_replication_date} ne $TIMESTAMP) {
-        warn "Applied changes, but last_replication_date is '$row->{last_replication_date}' not '$TIMESTAMP'\n";
+    # If we're applying a previously aborted sequence, $REPLICATION_SEQUENCE
+    # and $TIMESTAMP will not be defined.
+    if (defined $aborted_sequence) {
+        $aborted_sequence = undef;
+    } else {
```

Comment:
> Should this special case be logged?

We log "Continuing a previously aborted load" at the start, but if you mean another log entry here when the aborted load has finished, then it certainly can't hurt.

> When resuming aborted load, what guarantee of integrity do we have on the data previously loaded and the data to be loaded?

The changes inside the packet are grouped by transaction ID, so `ProcessReplicationChanges` applies each group in a separate transaction (as they were on the master). The corresponding pending_data entries are deleted in the same transaction that they're applied in, so they're only removed once they're applied (i.e. a transaction cannot be half-applied if the script is aborted). This code is decades old, so it's fairly well-tested.

Reply:
Thanks, I did ask because it was unclear from the commit message from which point it would “attempt to continue applying any existing data that was loaded”. It might be worth reminding when the transaction is taking part in the commit message to make it clear where the “existing” pending data is coming from and why it’s guaranteed to be totally unapplied and complete.

Reply:
Added a log message (in commit "Log success after applying a previously aborted load") and updated the commit message for "LoadReplicationChanges: Don't error after applying an aborted sequence."